src/client/pages/Offer/Introduction/ExternalInsuranceProvider/Compare.tsx

```diff
@@ -70,15 +72,36 @@ const CompareBoxDescription = styled.div({
   fontSize: '0.875rem',
   marginTop: '0.5rem',
   color: '#505050',
+
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '0.25rem',
 })
 
 interface Props {
   insuranceDataCollection: InsuranceDataCollection
   cost: InsuranceCost
+  description?: string
 }
 
-export const Compare: React.FC<Props> = ({ insuranceDataCollection, cost }) => {
+export const Compare: React.FC<Props> = (props) => {
+  const textKeys = useTextKeys()
+  const { insuranceDataCollection, cost, description } = props
   const translateCoverage = useTranslateCoverage()
+  const getDateLabel = useGetDateLabel()
+
+  const compareDescriptions = [
```

Comment:
Nit: log of magic to build an array conditionally.  Could be simplified as 
```
const compareDesciptions = []
if (insuranceDataCollection.renewalDate) {
  compareDescriptions.push(...)
}
if (insuranceDataCollection.coverage) {
  compareDescriptions.push(...)
}
```

Reply:
fair enough :)